spark3/src/test/java/org/apache/iceberg/spark/sql/TestDropTable.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.spark.sql;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Map;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.spark.SparkCatalog;
+import org.apache.iceberg.spark.SparkCatalogTestBase;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+
+public class TestDropTable extends SparkCatalogTestBase {
+  private final boolean shouldPurgeDataAndMetadata;
+
+  public TestDropTable(String catalogName, String implementation, Map<String, String> config) {
+    super(catalogName, implementation, config);
+    if (config.containsKey(SparkCatalog.PURGE_DATA_AND_METADATA)) {
+      this.shouldPurgeDataAndMetadata = Boolean.parseBoolean(config.get(SparkCatalog.PURGE_DATA_AND_METADATA));
+    } else {
+      this.shouldPurgeDataAndMetadata = SparkCatalog.PURGE_DATA_AND_METADATA_DEFAULT;
+    }
+  }
+
+  @Parameterized.Parameters(name = "catalogName = {0}, implementation = {1}, config = {2}")
+  public static Object[][] parameters() {
+    return new Object[][]{
+            {"testhive", SparkCatalog.class.getName(),
+                    ImmutableMap.of(
+                            "type", "hive",
+                            "default-namespace", "default"
+                    )},
+            {"testhadoop", SparkCatalog.class.getName(),
+                    ImmutableMap.of(
+                            "type", "hadoop"
+                    )},
+            {"testhive_purge", SparkCatalog.class.getName(),
+                    ImmutableMap.of(
+                            "type", "hive",
+                            "default-namespace", "default",
+                            SparkCatalog.PURGE_DATA_AND_METADATA, "false"
+                    )}
+    };
+  }
```

Comment:
Nit - These appear over indented.

There should be other examples that have the correct formatting. Maybe take a look at `TestSparkCatalogHadoopOverrides`, which has a very similar set of parameters (with proper spacing).

Reply:
Thanks, I will update formatting

Reply:
done